.github/workflows/build.yml

```diff
@@ -222,25 +222,15 @@ jobs:
       - name: Install meson and ninja
         run: pip install meson ninja
 
-      - name: Set up environment variables
-        run: |
-          "LUA_SUBPROJECT_PATH=subprojects/$(awk -F ' *= *' '/directory/ { printf $2 }' subprojects/lua.wrap)" >> $env:GITHUB_ENV
-
-      - name: Download subprojects
-        shell: bash
-        run: |
-          meson subprojects download
-
       - name: Configure
         run: meson setup --wrap-mode=forcefallback build
 
       - name: Build
-        run: meson install -C build --destdir="../lite-xl"
+        run: meson install -C build --skip-subprojects --destdir="../lite-xl"
```

Comment:
Should subprojects be skipped?

Reply:
until we have proper install tags and subproject exclusion in place.